templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/CategorySearch/product_grid_panel.html.twig

```diff
@@ -0,0 +1,59 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% set displayName = true %}
+{% if (grid.view_options.display_name is defined) %}
+  {% set displayName = grid.view_options.display_name %}
+{% endif %}
+
+<div class="card js-grid-panel" id="{{ grid.id }}_grid_panel">
```

Comment:
I don't think you need to copy the whole file Instead you could extend the original `grid_panel.html.twig` template and only override the part you need (which should be the `grid_actions_block` block)

Reply:
Oh actually I thought the category filter was with the bulk actions, but it seems to be more a part of the grid table itself Then in this template you need to override the `grid_view_block` so that you can include your custom product grid which includes category filtering